src/section/mod.rs

```diff
@@ -150,8 +153,14 @@ impl Section {
         self.members.update(member_info)
     }
 
-    pub fn to_minimal(&self) -> Self {
-        let first_key_index = self.elders_info_signing_key_index();
+    // Returns a trimmed version of this `Section` which contains only the elders info and the
+    // section chain truncated to the given length (the chain is truncated from the end, so it
+    // always contains the latest key). If `chain_len` is zero, it is silently replaced with one.
```

Comment:
if `chain_len > existing_len`, the whole chain will be returned, right?

Reply:
Yes.